README.md

```diff
@@ -14,10 +14,10 @@ The current version of this code is optimized for x86_64 machine architecture, b
 
 # Collection
 
-The Synch framework provides a large set of highly efficient concurrent data-structures, such as combining-objects, concurrent queues and stacks, concurrent hash-tables and locks. The cornerstone of the Synch framework are the combining objects. A Combining object is a concurrent object/data-structure that is able to simulate any other concurrent object, e.g. stacks, queues, atomic counters, barriers, etc. The Synch framework provides the PSim wait-free combining object [2,10], the blocking combining objects CC-Synch, DSM-Synch and H-Synch [1], and the blocking combining object based on the technique presented in [4]. Moreover, the Synch framework provides the Osci blocking, combining technique [3] that achieves good performance using user-level threads.
+The Synch framework provides a large set of highly efficient concurrent data-structures, such as combining-objects, concurrent queues and stacks, concurrent hash-tables and locks. The cornerstone of the Synch framework are the combining objects. A Combining object is a concurrent object/data-structure that is able to simulate any other concurrent object, e.g. stacks, queues, atomic counters, barriers, etc. The Synch framework provides the PSim wait-free combining object [2,10], the blocking combining objects CC-Synch, DSM-Synch and H-Synch [1], and the blocking combining object based on the technique presented in [4]. Moreover, the Synch framework provides the Osci blocking, combining technique [3] that achieves good performance using user-level threads. Since v3.1.0, the Synch framework provides another implementation of the flat-combining synchronization technique [14] that is written from the scratch and it has no relation with the original code of flat-combining provided in [15]. In many cases, this new implementation of flat-combining performs much better than the original implementation [15].
```

Comment:
Maybe:
The Synch framework offers an implementation of the flat-combining synchronization technique [14] as provided in [15]. Since v3.1.0, a second more performant implementation of flat-combining synchronization is provided. This new version is implemented from from the scratch and is not just an optimized version of the original code.

Reply:
The Synch framework did not provided any implementation of of flat-combining! With this patch, it would provide a new alternative implementation of the flat-combining written from the scratch.

Reply:
Then maybe:
Since v3.1.0, the Synch framework offers a new performant implementation of flat-combining synchronization technique [14]. This novel version is implemented from the scratch and is not just an optimized version of the original code described in [15].

Reply:
Done.